docs/api/portable-stories/portable-stories-playwright.mdx

```diff
@@ -61,7 +61,7 @@ sidebar:
 
   (⚠️ **Experimental**)
 
-  Instead of using Playwright's own `test` function, you can use Storybook's special `createTest` function to [extend Playwright's base fixture](https://playwright.dev/docs/test-fixtures#creating-a-fixture) and override the `mount` function to load, render, and play the story. This function is experimental and is subject to changes.
+  Instead of using Playwright's own `test` function, you can use Storybook's special `createTest` function to [extend Playwright's base fixture](https://playwright.dev/docs/test-fixtures#creating-a-fixture) and override the `mount` function to load, render, and run the story. This function is experimental and is subject to changes.
```

Comment:
I think this should still be play, what does run mean for people using the PW api? the run method is used internally. This should just say that it calls the play function